src/shared/geometry.ts

```diff
@@ -577,7 +609,7 @@ export function grabHeatmapData(
   logType: string,
   timeRange: "enabled" | "auto" | "teleop" | "teleop-no-endgame" | "full",
   uuid?: string,
-  numberArrayFormat?: "Translation2d" | "Translation3d" | "Pose2d" | "Pose3d",
+  numberArrayFormat?: "Translation2d" | "Translation3d" | "Pose2d" | "Pose3d" | "DifferentialSample" | "SwerveSample",
```

Comment:
I don't think this makes sense to support for heatmap data but I get type errors if I don't add this

Reply:
You don't want to add the types anywhere related to reading number arrays, including `grabNumberArray` (that's only used for the legacy formats). As long as the struct format is supported in `grabPosesAuto` it should work as expected.